core/neat/mixins/_grid-column.scss

```diff
@@ -0,0 +1,38 @@
+/// Creates Neat a grid column of requested size.
+///
+/// @argument {number} $width
+///
+/// @argument {map} $grid
+///   The type of grid for this column. By default, the global
```

Comment:
Maybe we mean something like “the grid parameters to be used”, or something like that?